channels/ae.m3u

```diff
@@ -179,6 +179,9 @@ http://51.158.73.206/hls/stream.m3u8
 http://livecdnh2.tvanywhere.ae/hls/wanasah/04.m3u8
 #EXTINF:-1 tvg-id="Dubai One" tvg-name="Dubai One" tvg-logo="https://i.imgur.com/WjJ8MHC.png" group-title="",Dubai One
 http://barakatiptv.com:80/live/JC65XhKF5j/gG1rNsDNGf/69952.m3u8
-
-
-
+#EXTINF:-1 tvg-id="Dubai One" tvg-name="NatGeo People" tvg-logo="https://i.imgur.com/7BMvcQx.png" group-title="",NatGeo People
+http://livecdnh1.tvanywhere.ae:80/hls/nat_geo_people/index.m3u8
+#EXTINF:-1 tvg-id="Star Movies ARB" tvg-name="Star Movies ARB" tvg-logo="https://i.imgur.com/DQ1RfUa.png" group-title="Movies",Star Movies HD
+http://livecdnh1.tvanywhere.ae:80/hls/star_movies/index.m3u8
+#EXTINF:-1 tvg-id="NatGeo Wild" tvg-name="NatGeo Wild" tvg-logo="https://i.imgur.com/zQv0hZU.png" group-title="Movies",NatGeo Wild
+http://livecdnh1.tvanywhere.ae:80/hls/nat_geo_wild/index.m3u8
```

Comment:
https://www.lyngsat.com/tvchannels/us/National-Geographic-Wild.html

Link works , but must be placed in us.m3u playlist